test/specs/repo/extensions/nuke.ts

```diff
@@ -38,16 +38,12 @@ describe( 'Nuke', function () {
 
 		await browser.acceptAlert();
 
-		await $( 'li*=has been queued for deletion' );
-
-		await browser.waitForJobs();
```

Comment:
The test was test was split into two, but the same thing is being tested. The "confirm that was actually deleted" part had to be moved to a separate test so it could have `.retries` applied as it was being flakey now that things are running in Job Runner.

Reply:
Got it. My impression is, that relying on retries instead of explicitly waiting for jobs to be finished is quite implicit and hard to understand intuitively. Not a blocker for the PR though.

Reply:
Of course I agree, I can say more :)

Despite that I think this becoming flakey somehow had to do with the Job Runner running instead of jobs on request, `waitForJobs` then checking was still not giving a consistent result. I had watched it headed and spent a couple hours going over different options before I landed on this. It simply is not that predictable when the page will disappear and we don't seem to have a way to track when it's done, so it came down to retires or a long `browser.pause()`, and I chose the later.  `retries` I think is the better of those two options, and I think everywhere else it is applied currently in our code is the same situation: there wasn't another clear way and it didn't come down to jobs clearing the queue...